src/api/profile/follow.js

```diff
@@ -0,0 +1,11 @@
+import { accessInstance } from '../api';
+
+export const follow = async (accountName) => {
+  const res = await accessInstance.post(`/profile/${accountName}/follow`);
+  return res.data;
+};
+
+export const unfollow = async (accountName) => {
```

Comment:
```suggestion
export const deleteFollow = async (accountName) => {
```
요청 방식 + 기능의 형태로 함수 명 수정해주시고, 폴더 생성 후 두 가지 파일로 나눠주셔도 좋을 것 같습니다. 근데 저도 이 부분이 잘 안 되고 있어서, 리팩토링 때 한 번 전체적으로 수정해도 됩니다!